packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/ChipGroupDemo/ChipGroupDemo.tsx

```diff
@@ -1,14 +1,27 @@
 import { Badge, Chip, ChipGroup } from '@patternfly/react-core';
 import React, { Component } from 'react';
 
-export interface BadgeChipProps extends React.HTMLProps<HTMLDivElement> {
-    children?: React.ReactNode; 
-    isRead?: boolean; 
-    className?: string; 
+export interface BadgeChipState {
```

Comment:
This should be fixed in react-core and not exported in ChipGroupDemo.  The issue here is that BadgeChipProps is not being exported in react-core.